bal_schema_v1.5.json

```
@@ -0,0 +1,205 @@
+{
+    "$schema": "https://specs.frictionlessdata.io/schemas/table-schema.json",
+    "name": "base_adresse_locale",
+    "title": "Base Adresse Locale",
+    "description": "Spécification du fichier d'échange visant à diffuser des données voies-adresses locales",
+    "keywords": [
+        "adresse",
+        "voie"
+    ],
+    "countryCode": "FR",
+    "homepage": "https://github.com/aitf-sig-topo/voies-adresses",
+    "path": "https://github.com/aitf-sig-topo/voies-adresses/schemas/v1.5/bal_schema_v1.5.json",
+    "licenses": [{
+            "title": "CC BY-NC-ND 4.0",
+            "name": "CC BY-NC-ND 4.0",
+            "path": "https://creativecommons.org/licenses/by-nc-nd/4.0/deed.fr"
+        }
+    ],
+    "created": "2025-03-25",
+    "lastModified": "2025-11-14",
+    "version": "v1.5.0",
+    "contributors": [{
+            "title": "Groupe de travail SIG et topographie",
+            "organisation": "AITF - Association des Ingénieur.e.s territoriaux et Ingénieur.e.s en chef territoriaux de France",
+            "role": "author"
+        }
+    ],
+    "fields": [{
+            "name": "id_ban_commune",
+            "description": "identifiant unique de la commune fourni par la BAN",
+            "type": "string",
+            "format": "uuid",
```

- sur les 3 id_ban 
- les "constraints/ required" doivent passer à "True"
- utiliser un pattern pour préciser que la version du format uuid attendue est la version uuidv4. {
  "type": "string",
  "format": "uuid",
  "pattern": "^[0-9a-f]{8}-[0-9a-f]{4}-4[0-9a-f]{3}-[89ab][0-9a-f]{3}-[0-9a-f]{12}$"
}
```suggestion
            "format": "uuid",
            "pattern": "^[0-9a-f]{8}-[0-9a-f]{4}-4[0-9a-f]{3}-[89ab][0-9a-f]{3}-[0-9a-f]{12}$",
```

---

voici la confirmation que UUID en data type implique un test implicite : 

Erreur à la ligne N/A: Type error in the cell "1c62879e-9b47-4364-af13-5b8d1105" in row "26" and field "id_ban_toponyme" at position "2": type is "string/uuid"